tests/datasource/fluent/test_spark_filesystem_datasource.py

```diff
@@ -83,6 +83,25 @@ def test_add_csv_asset_to_datasource(
     assert m1 is not None
 
 
+# TODO: Parametrize with all asset types
```

Comment:
@anthonyburdi Do we need to add a test for other storage possibilities (e.g., S3, etc.), or is this sufficient?

Reply:
I'm not sure yet, I'll be doing a deeper dive on testing as I address the other assets. My gut says no, that we should test accessing other storage possibilities separately rather than for each asset.